client/app/scripts/superdesk/editor/views/add-embed.html

```diff
@@ -0,0 +1,25 @@
+<div class="add-embed__plus" ng-if='!vm.extended' ng-click="vm.toggle()" sd-text-editor-drop-zone>
+    <i class="icon-plus-small icon-round icon-white"></i>
+    <div class="add-embed__drop-zone"></div>
+</div>
+<div class="add-embed__body" ng-if='vm.extended'>
+    <form class="form-inline">
+        <div class="form-group">
+            <input type="text"
+                   class="line-input"
+                   placeholder="Past url or embed code"
+                   ng-change="vm.updatePreview()"
+                   ng-model="vm.input"/>
+        </div>
+        <div class="form-group">
+            <button type="reset" class="btn btn-small btn-default" ng-click="vm.toggle()">
+                <i class="icon-close-small"></i>
+            </button>
+            <button type="submit" class="btn btn-small btn-info" ng-click="vm.createBlockFromEmbed()">
+                <i class="icon-white icon-ok"></i>
+            </button>
+        </div>
+        Preview:
```

Comment:
Texts should be translated, a canonical pattern looks like this:
```html
{{:: 'Preview:' | translate }}
```
(basically using the `translate` filter to do the heavy lifting)